### DISCUSSION TITLE
Export without pattern / grid

### DISCUSSION THREAD

**Discussion by torgeros** [flxzt/rnote#258] [Q&A]
I like grid lines on my paper as an orientation for how big I should write/draw when zoomed in so that the size is good when the pdf is opened in whatever pdf reader. In there, however, I do no longer need the grid lines. Is there an option to export the canvas to a pdf without the pattern?

**Comment by flxzt** [comment]
Hi, it is possible. Just disable the pattern temporarily in the settings before exporting.  
![Bildschirmfoto vom 2022-09-25 13-10-34](https://user-images.githubusercontent.com/19841886/192140567-9df02317-d18e-42b6-b5be-1ce1fb4f831e.png)

**Reply by torgeros** [reply]
Yeah I figured that, but this is not a practical way, is it?

I have no plan of rust so I will not be able to implement it myself, but would it be worth to open a FR for this?

**Reply by flxzt** [reply]
Sorry for the late answer. I opened a issue for it myself in #319 .
